types/src/on_chain_config/aptos_features.rs

```diff
@@ -141,6 +142,7 @@ impl FeatureFlag {
             FeatureFlag::WEBAUTHN_SIGNATURE,
             // FeatureFlag::RECONFIGURE_WITH_DKG, //TODO: re-enable once randomness is ready.
             FeatureFlag::KEYLESS_ACCOUNTS,
+            //FeatureFlag::FEDERATED_KEYLESS, // disabled by default
```

Comment:
why disabled by default?

Reply:
you probably want this to be enabled by default and see if any existing functionalities are affected?

Reply:
Oh, I guess this will be for devnet, right?

Reply:
yup

Reply:
okay, will add